float32 latitude
float32 longitude
string class_name
string file_name